client/my-sites/patterns/hooks/use-render-readymade-template.ts

```diff
@@ -1,23 +1,41 @@
-import { RenderedPattern } from '@automattic/block-renderer';
 import { useQuery } from '@tanstack/react-query';
 import wpcom from 'calypso/lib/wp';
 import { RENDERER_SITE_ID } from 'calypso/my-sites/patterns/constants';
 import type { ReadymadeTemplate } from 'calypso/my-sites/patterns/types';
 
+// Shouldnt have to do this.
```

Comment:
Right, we shouldn't need this. The API already returns _slugified_ style variations. Why do we need this?

Reply:
Not sure. Maybe Dean was getting a stale payload from the API. 🤷 

I've removed it anyway.